danish/haervejen/haervejen.md

```diff
@@ -1,4 +1,4 @@
-Du vandrer sydpå og fra bakkerne hører du lyden af [musik](https://www.youtube.com/watch?v=cUGd8X7bjuo).
+Du vandrer sydpå og fra bakkerne hører du lyden af "Bob bob bob". Det er Benny, som vandrer rundt og snakker med sig selv.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.